x/dispensation/types/records.go

```diff
@@ -25,19 +26,20 @@ func (d ClaimStatus) String() string {
 }
 
 // DistributionRecord is created for every recipient for a distribution
-// TODO : Add ClaimStatus to the prefixed key for records
+// TODO : Add DistributionStatus to the prefixed key for records
```

Comment:
Should it be addressed as part of this PR?

Reply:
Same as above , ill make these changes in the .42 version